Firmware/RTK_Surveyor/Display.ino

```diff
@@ -477,7 +477,10 @@ void displaySplash()
   if (online.display == true)
   {
     //Display SparkFun Logo for at least 1/10 of a second
-    while ((millis() - splashStart) < 100)
+    unsigned long minSplashFor = 100;
+    if (productVariant == REFERENCE_STATION) // Reference station starts up very quickly. Keep splash on for longer
```

Comment:
If there's anything you've learned or can do to improve the start up time of non-reference stations, please let me know.